Code/Editor/TrackView/SequenceBatchRenderDialog.cpp

```diff
@@ -958,40 +936,30 @@ void CSequenceBatchRenderDialog::CaptureItemStart()
     int i = 2;
     while (QFileInfo::exists(finalFolder))
     {
-        finalFolder = folder;
-        const QString suffix = QString::fromLatin1("_v%1").arg(i);
-        finalFolder += suffix;
+        finalFolder = folder + QString::fromLatin1("_v%1").arg(i);
         ++i;
     }
 
-    // create a new folder before writing any files
-    QDir().mkdir(finalFolder);
+    // Create a new folder before writing any files
+    [[maybe_unused]] const bool mkdirResult = QDir().mkdir(finalFolder);
+    AZ_Assert(mkdirResult, "Cannot create directory %s", finalFolder.toStdString().c_str());
```

Comment:
maybe return directly, and open a popup to the user to state the issue as it has to be noticed (I'm not sure of the assert policy of o3de, if it crashes or if they are skippable). MessagePopupRequests seems to does that but does not seems used much so maybe there is something even newer to show error messages as popup